gradle/documentation/render-javadoc.gradle

```diff
@@ -325,6 +342,7 @@ class RenderJavadocTask extends DefaultTask {
     // lucene-test-framework was first, or broken links to things like LuceneTestCase if lucene-core was first)
     if (project.path != ':solr:test-framework') {  //
       findRenderTasksInDependencies()
+          .findAll({it.project.path != ':solr:solrj' && it.project.path != ':solr:solrj-zookeeper'}) // split packages
```

Comment:
Why is this needed? I know it mentions split packages - but don't we have split packages elsewhere and those modules aren't listed here?

Reply:
The link checker of our docs complains about split packages here but perhaps it didn't for other split packages cases?  This is possible; it depends on if the dependency reflects itself in the public API.  If there is a compile dependency that isn't reflected in the API, then there is no javadoc issue.